fdbclient/include/fdbclient/CommitTransaction.h

```diff
@@ -274,6 +274,7 @@ struct CommitTransactionRef {
 	// usual commit path. It is currently only used during backup mutation log restores.
 	VectorRef<Optional<MutationRef>> encryptedMutations;
 	Version read_snapshot = 0;
+	bool bypass_storage_quota = false;
```

Comment:
To save space, this information can be stored in `CommitTransactionRequest::flags` instead.